.github/workflows/deploy.yml

```
@@ -0,0 +1,34 @@
+name: deploy_jenkins
+on: 
+  workflow_dispatch: []
```

nit but it can just be blank
```suggestion
  workflow_dispatch:
```